internal/pkg/cli/env_show.go

```diff
@@ -1,15 +1,25 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-
 package cli
 
 import (
 	"fmt"
+	"io"
+
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 	"github.com/spf13/cobra"
 )
 
+const (
+	environmentShowProjectNamePrompt     = "Which project's environments would you like to show?"
+	environmentShowProjectNameHelpPrompt = "A project groups all of your applications together."
+	environmentShowEnvNamePrompt         = "Which environment of %s would you like to show?"
```

Comment:
nit: We usually prefix string constants that contain format verbs like "%s" with `fmt`.   
So something like: `fmtEnvShowNamePrompt` this makes it easy to notice that the string itself shouldn't be printed as is but needs some data.

Reply:
Thanks! Made the change here and in test file.